wordpress-mobile/WordPress-Android
# Fix #5372: Account for 'false' api_blogid in migration

Fixes #5372. `api_blogid` can be `false` for self-hosted sites in some cases. This both causes a crash if there are drafts for those sites, and causes those sites not to be migrated at all.

To test:
1. Check out 0991171, run tests in `FluxCMigrationTest` 💥
2. Check out branch, run tests 👍
